LanguageFeatures/Augmentation-libraries/type_inheritance_A02_t03.dart

```diff
@@ -2,94 +2,90 @@
 num topLevelFunction() => 0;
 
 class C {
   static num get staticGetter => 0;
-  static num staticMethod() => 0;
-  num get instanceGetter => 0;
+  static num staticMethod();
+  num get instanceGetter;
   num instanceMethod() => 0;
 }
 
 mixin M {
   static num get staticGetter => 0;
-  static num staticMethod() => 0;
-  num get instanceGetter => 0;
+  static num staticMethod();
+  num get instanceGetter;
   num instanceMethod() => 0;
 }
 
 enum E {
   e0;
   static num get staticGetter => 0;
-  static num staticMethod() => 0;
-  num get instanceGetter => 0;
+  static num staticMethod();
+  num get instanceGetter;
   num instanceMethod() => 0;
 }
 
 class A {}
 
 extension Ext on A {
   static num get staticGetter => 0;
-  static num staticMethod() => 0;
-  num get instanceGetter => 0;
+  static num staticMethod();
+  num get instanceGetter;
   num instanceMethod() => 0;
 }
 
 extension type ET(num id) {
   static num get staticGetter => 0;
-  static num staticMethod() => 0;
-  num get instanceGetter => 0;
+  static num staticMethod();
+  num get instanceGetter;
   num instanceMethod() => 0;
 }
 
 class MA = Object with M;
 
 main() {
   Expect.equals(1, topLevelGetter);
-  Expect.equals(2, topLevelFunction());
-  Expect.equals(3, C.staticGetter);
-  Expect.equals(4, C.staticMethod());
-  Expect.equals(5, C().instanceGetter);
-  Expect.equals(6, C().instanceMethod());
-  Expect.equals(7, M.staticGetter);
-  Expect.equals(8, M.staticMethod());
-  Expect.equals(9, MA().instanceGetter);
-  Expect.equals(10, MA().instanceMethod());
-  Expect.equals(11, E.staticGetter);
-  Expect.equals(12, E.staticMethod());
-  Expect.equals(13, E.e0.instanceGetter);
-  Expect.equals(14, E.e0.instanceMethod());
-  Expect.equals(15, Ext.staticGetter);
-  Expect.equals(16, Ext.staticMethod());
-  Expect.equals(17, A().instanceGetter);
-  Expect.equals(18, A().instanceMethod());
-  Expect.equals(19, ET.staticGetter);
-  Expect.equals(20, ET.staticMethod());
-  Expect.equals(21, ET(0).instanceGetter);
-  Expect.equals(22, ET(0).instanceMethod());
-  Expect.equals(23, ET(0).id);
+  Expect.equals(0, topLevelFunction());
+  Expect.equals(0, C.staticGetter);
+  Expect.equals(1, C.staticMethod());
+  Expect.equals(1, C().instanceGetter);
+  Expect.equals(0, C().instanceMethod());
+  Expect.equals(0, M.staticGetter);
+  Expect.equals(1, M.staticMethod());
+  Expect.equals(1, MA().instanceGetter);
+  Expect.equals(0, MA().instanceMethod());
+  Expect.equals(0, E.staticGetter);
+  Expect.equals(1, E.staticMethod());
+  Expect.equals(1, E.e0.instanceGetter);
+  Expect.equals(0, E.e0.instanceMethod());
+  Expect.equals(0, Ext.staticGetter);
+  Expect.equals(1, Ext.staticMethod());
+  Expect.equals(1, A().instanceGetter);
+  Expect.equals(0, A().instanceMethod());
+  Expect.equals(0, ET.staticGetter);
+  Expect.equals(1, ET.staticMethod());
+  Expect.equals(1, ET(0).instanceGetter);
+  Expect.equals(0, ET(0).instanceMethod());
+  Expect.equals(42, ET(42).id);
```

Comment:
Considering the comment on LanguageFeatures/Augmentation-libraries/type_inheritance_A02_t02.dart, perhaps there shouldn't be an `id` case here, either?

Reply:
Or perhaps it's the other way around and an `id` case should be added to LanguageFeatures/Augmentation-libraries/type_inheritance_A02_t02.dart after all?

Reply:
Yes, has been added already :)